custom_components/mail_and_packages/const.py

```diff
@@ -202,6 +237,114 @@
     },
     "royal_packages": {},
     "royal_tracking": {"pattern": ["[A-Za-z]{2}[0-9]{9}GB"]},
+    # Poczta Polska SA
+    "poczta_polska_delivered": {},
```

Comment:
I'd like to try to keep these names simple like `pocztapolska_delivered` and `inpostpl_delivered`

Reply:
The issue with this is that these IDs are generated from names in `SensorEntityDescription`. 
`Mail InPost.pl Delivered` then becomes `mail_inpost_pl_delivered` and so on.

I suspect there will be more cases like this in the future, where shipper's name contain multiple words.